test/db/formats/coff

```diff
@@ -160,15 +161,14 @@ EOF
 EXPECT=<<EOF
 |           0x00008dbd      e80e000000     call  sym._TIFFVGetField    ; RELOC 32 _TIFFVGetField @ 0x00008dd0
 sym._TIFFGetField 0x8dbd [CALL] call sym._TIFFVGetField
-|           0x00008dde      e8ad280000     call  sym.imp._TIFFFindField; RELOC 32 _TIFFFindField
+|           0x00008dde      e8d5280000     call  sym.imp._TIFFFindField; RELOC 32 _TIFFFindField
```

Comment:
Why is this difference? It wasn't patched before?

Reply:
Both test results are patched, but the artificial offset is a bit different now, so both are correct.

Reply:
you mean that is mapped to a different offset?

Reply:
The reloc-target map may be mapped slightly different and the assigned offsets inside of it may be different too.